epoxy-adapter/src/main/java/com/airbnb/epoxy/stickyheader/HasStickyHeader.kt

```diff
@@ -0,0 +1,39 @@
+package com.airbnb.epoxy.stickyheader
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+
+/**
+ * Adds sticky headers capabilities to any [RecyclerView.Adapter]
+ * combined with [StickyHeaderLinearLayoutManager].
+ */
+interface HasStickyHeader {
```

Comment:
I think `HasStickyHeader` naming doesn't feel quite right, since this interface does a bit more than that. maybe something like `StickyHeaderCallbacks`

Reply:
Yeah it's not the name which I would prefer as first choice. `StickyHeaderCallbacks` seem good to me, any other suggestion which come to your mind?

Reply:
nope, I think that's great

Reply:
So should I keep it `HasStickyHeader`?

Reply:
sorry, I meant `StickyHeaderCallbacks` seems good